entrypoint/dualstack/dualstackcluster_test.go

```diff
@@ -0,0 +1,126 @@
+package dualstack
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test:", func() {
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Validate Dual-Stack Ingress Service", func() {
+		_, err := shared.ManageWorkload("apply", "dualstack-ingress.yaml")
+		Expect(err).NotTo(HaveOccurred())
+		
+		testdata := map[string]string{
+			"namespace":"default",
+			"label":"app=dualstack-ing",
+			"svc":"dualstack-ing-svc",
+			"expected":"dualstack-ing-ds", 
+		}
+		
+		testcase.TestPodStatusIsRunningByNamespaceAndLabel(testdata["namespace"],testdata["label"])
+		testcase.TestIngressDualStack(testdata)
+
+		_, err = shared.ManageWorkload("delete", "dualstack-ingress.yaml")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("Validate Dual-Stack NodePort Service", func() {
+		_, err := shared.ManageWorkload("apply", "dualstack-nodeport.yaml")
+		Expect(err).NotTo(HaveOccurred())
+
+		testdata := map[string]string{
+			"namespace":"default",
+			"label":"app=dualstack-nodeport",
```

Comment:
all testcases functionalities must comply the same signature 
https://github.com/rancher/distros-test-framework/blob/2ce312df8081eb1d6a8d2043cf9013afe75daa76/pkg/customflag/model.go#L50C5-L50C5

wich is only deploy workload as param.